modules/ROOT/pages/amazon/amazon-ec2-connector.adoc

```diff
@@ -71,48 +87,46 @@ In the image above, the placeholder values refer to a configuration file `mule-a
 `src/main/resources` folder of your project.
 You can either enter your credentials into the global configuration properties, or reference a configuration file that contains these values. For simpler maintenance and better re-usability of your project, Mule recommends that you use a configuration file. Keeping these values in a separate file is useful if you need to deploy to different environments, such as production, development, and QA, where your access credentials differ.
```

Comment:
Although you could enter your credentials into the global configuration properties, use a configuration file in case you need to deploy to different environments, such as production, development, and QA, where your access credentials differ.